sdks/java/io/hadoop-input-format/src/test/java/org/apache/beam/sdk/io/hadoop/inputformat/HIFIOWithEmbeddedCassandraTest.java

```diff
@@ -139,7 +139,7 @@ public Configuration getConfiguration() {
     conf.setClass("mapreduce.job.inputformat.class",
         org.apache.cassandra.hadoop.cql3.CqlInputFormat.class, InputFormat.class);
     conf.setClass("key.class", java.lang.Long.class, Object.class);
-    conf.setClass("value.class", com.datastax.driver.core.Row.class, Object.class);
+    conf.setClass("value.class", Row.class, Object.class);
```

Comment:
These qualified `Row` are better to be fully qualified, since `Row` mostly should be a Beam Row. But it is OK here because it would probably be a pain to get an IDE to do the right thing.